examples/redirect_health_group/main.tf

```diff
@@ -0,0 +1,24 @@
+terraform {
+  required_providers {
+    aci = {
+      source = "ciscodevnet/aci"
+    }
+  }
+}
+
+provider "aci" {
+  username = "" # <APIC username>
+  password = "" # <APIC pwd>
+  url      = "" # <cloud APIC URL>
+  insecure = true
+}
+
+resource "aci_tenant" "terraform_tenant" {
+  name        = "tf_tenant"
+  description = "This tenant is created by terraform"
+}
+
+resource "aci_l4_l7_redirect_health_group" "example" {
+  tenant_dn  = aci_tenant.terraform_tenant.id
+  name       = "example"
+}
```

Comment:
Can you also add this section in the example of aci_destination_of_redirected_traffic ?